go/gregor/client/client.go

```diff
@@ -6,31 +6,31 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/keybase/client/go/logger"
+
 	"github.com/keybase/client/go/gregor"
 	"github.com/keybase/client/go/protocol/chat1"
 	"github.com/keybase/client/go/protocol/gregor1"
 	"golang.org/x/net/context"
-
-	"github.com/keybase/go-framed-msgpack-rpc/rpc"
 )
 
 type LocalStorageEngine interface {
-	Store(gregor.UID, []byte) error
-	Load(gregor.UID) ([]byte, error)
+	Store(gregor.UID, []byte, [][]byte) error
+	Load(gregor.UID) ([]byte, [][]byte, error)
```

Comment:
why this interface of `[][]bytes`?  Maybe some more comments as to why it's so generic?

Reply:
Same reason the state is `[]byte`. Marshaling happens one level up.

Reply:
I think ultimately the reason why here is that all these Gregor types are interfaces, and the best thing you can get to store from those interfaces is bytes from `Bytes()`.